gym/utils/passive_env_checker.py

```diff
@@ -183,34 +183,23 @@ def env_reset_passive_checker(env, **kwargs):
                 f"Actual default: {seed_param}"
             )
 
-    if "return_info" not in signature.parameters and not (
-        "kwargs" in signature.parameters
-        and signature.parameters["kwargs"].kind is inspect.Parameter.VAR_KEYWORD
-    ):
-        logger.warn(
-            "Future gym versions will require that `Env.reset` can be passed `return_info` to return information from the environment resetting."
-        )
-
-    if "options" not in signature.parameters and "kwargs" not in signature.parameters:
+    if "options" not in signature.parameters or "kwargs" in signature.parameters:
         logger.warn(
             "Future gym versions will require that `Env.reset` can be passed `options` to allow the environment initialisation to be passed additional information."
         )
 
     # Checks the result of env.reset with kwargs
     result = env.reset(**kwargs)
-    if kwargs.get("return_info", False) is True:
```

Comment:
Personally, I would like the passive environment checker to have as minimal errors as possible. Therefore, keep some of these checks as warnings that the results is tuple and length == 2

Reply:
I made the tuple check into a warning.